src/features/ajax/aggregate/index.js

```diff
@@ -119,6 +120,9 @@ export class Aggregate extends AggregateBase {
         event.spanTimestamp = xhrContext.dt.timestamp
       }
 
+      // parsed from the AJAX body, looking for operationName param & parsing query for operationType
+      event.gql = params.gql = parseBatchGQL(parseGQLContents(this.body)) || parseGQL(parseGQLQueryString(this?.parsedOrigin?.search))
```

Comment:
Could we simplify this by having a single function we call passing the body and search string and assuming the gql module will take care of figuring out whether to use body or search string and whether batching is being used?

Reply:
yeah, good idea